eng/pipelines/common/templates/runtimes/run-test-job.yml

```diff
@@ -20,6 +20,8 @@ parameters:
   pool: ''
   runtimeFlavor: 'coreclr'
   runtimeFlavorDisplayName: 'CoreCLR'
+  overrideDependsOn: false
```

Comment:
We don't need this extra parameter. We could just test if `dependsOn` has any value, i.e:
```yml
- ${{ if ne(parameters.dependsOn[0], '') }}:
```